packages/lesswrong/components/ea-forum/EADigestPage.tsx

```diff
@@ -277,22 +291,34 @@ const EADigestPage = ({ classes }: { classes: ClassesType<typeof styles> }) => {
       }
     </LWTooltip>
   }
+  
+  const startDate = digest?.startDate ? moment(digest.startDate).format('MMM D, YYYY') : null
+  const endDate = digest?.endDate ? moment(digest.endDate).format('MMM D, YYYY') : null
 
   return (
     <>
-      <HeadTags title={`EA Forum Digest #${digestNum}`} />
+      <HeadTags
+        title={`EA Forum Digest #${digestNum}`}
+        description={pageDescription}
+        image={digest?.onsiteImageId ? makeCloudinaryImageUrl(digest.onsiteImageId, socialImageCloudinaryProps) : undefined}
+      />
       <AnalyticsContext pageContext="DigestPage">
         <div className={classes.root} style={style}>
           <div className={classes.content}>
             <section className={classes.topSection}>
               <div>
+                {startDate && <div className={classes.dates}>
+                  <span className={classes.date}>{startDate}</span>
+                  <span>&mdash;</span>
```

Comment:
Is it always guaranteed that `startDate` and `endDate` are not null for a published digest? Otherwise we might want to hide the dash?